packages/auth/README.md

```diff
@@ -0,0 +1,51 @@
+[//]: # `{ "title": "@mintbase-js/auth", "order": 10 }`
+# @mintbase-js/auth
+
+This module abstracts authentication functions away from the Mintbase core SDK.
+
+As more complex use cases grow with credentials verification, signing, zero knowledge etc., having a stand alone library and documentation around these features seemed appropriate.
+
+# wallet.ts
+
+A stateful module that wraps the core features found in [near/wallet-selector](https://github.com/near/wallet-selector/).
+
+This module can be used as a framework agnostic wrapper using framework specific integrations:
+
+  * [React: WalletContext]($DOCS_BASE_URL/mintbase-sdk-ref//react#walletcontext)
```

Comment:
Still working on best way to do this. I think replacing paths from github -> gitbook is probably the better way to go vs the env vars